packages/create-invoice-form/src/lib/invoice/form.svelte

```diff
@@ -702,6 +702,20 @@
     font-size: 12px;
   }
 
+  :global(input:-webkit-autofill),
```

Comment:
![CleanShot 2024-11-19 at 15 04 18](https://github.com/user-attachments/assets/8857c67d-fcab-4ffd-89af-20c69060c292)

I'm still getting the color when I autofill.